Robot/robot.py

```diff
@@ -0,0 +1,38 @@
+from pygame import Vector2
+import pygame
+
+#definition of windowsize aufgrund fehlendes game state
+windowWidth = 640
+windowHeight = 480
+
+#robot class that defines a new player
+class Robot():
+    def __init__(self) -> None:
+        self.position = Vector2(320,240)
+        self.radius = 40
+        self.direction = Vector2(self.position.x,self.position.y + self.radius)
+        self.color = (0,0,255)
+        self.line = (self.position, Vector2(self.position,self.position.y + self.radius / 2))
```

Comment:
I find this definition rather hard to understand. I suggest changing `direction` to a number, perhaps `direction ∊ [0, 360)` and then drawing the line based on that?
At least `direction` doesn't sound like it should be tied to points in the coordinate plane.

Reply:
`direction ∊ (-180, 180]` now, because that's how pygame's polar coordinates work.